[2024-08-07T14:33:58.982+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T14:33:59.024+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow-space-exploration.Ver_data manual__2024-08-07T14:20:08.695612+00:00 [queued]>
[2024-08-07T14:33:59.045+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow-space-exploration.Ver_data manual__2024-08-07T14:20:08.695612+00:00 [queued]>
[2024-08-07T14:33:59.046+0000] {taskinstance.py:2306} INFO - Starting attempt 4 of 4
[2024-08-07T14:33:59.084+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): Ver_data> on 2024-08-07 14:20:08.695612+00:00
[2024-08-07T14:33:59.096+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=846) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-07T14:33:59.099+0000] {standard_task_runner.py:64} INFO - Started process 862 to run task
[2024-08-07T14:33:59.098+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '***-space-exploration', 'Ver_data', 'manual__2024-08-07T14:20:08.695612+00:00', '--job-id', '147', '--raw', '--subdir', 'DAGS_FOLDER/***-space-exploration.py', '--cfg-path', '/tmp/tmpwvw2gmrk']
[2024-08-07T14:33:59.100+0000] {standard_task_runner.py:91} INFO - Job 147: Subtask Ver_data
[2024-08-07T14:33:59.177+0000] {task_command.py:426} INFO - Running <TaskInstance: airflow-space-exploration.Ver_data manual__2024-08-07T14:20:08.695612+00:00 [running]> on host 38873943cc86
[2024-08-07T14:33:59.336+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***-space-exploration' AIRFLOW_CTX_TASK_ID='Ver_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-07T14:20:08.695612+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-07T14:20:08.695612+00:00'
[2024-08-07T14:33:59.340+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T14:33:59.382+0000] {logging_mixin.py:188} INFO - ls /tmp && head /tmp/platzi_data_20240807.csv
[2024-08-07T14:33:59.386+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-07T14:33:59.389+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T14:33:59.411+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=***-space-exploration, task_id=Ver_data, run_id=manual__2024-08-07T14:20:08.695612+00:00, execution_date=20240807T142008, start_date=20240807T143359, end_date=20240807T143359
[2024-08-07T14:33:59.479+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-07T14:33:59.517+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-07T14:33:59.532+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
